app/helpers/fhir_helper.rb

```diff
@@ -42,6 +42,7 @@ def patient_as_fhir(patient)
         }
       )],
       id: patient.id,
+      identifier: to_statelocal_identifier(patient.user_defined_id_statelocal),
```

Comment:
Since `identifier` is an array, would we want this to be:
```
[to_statelocal_identifier(patient.user_defined_id_statelocal)] 
```

It actually seems like the gem is smart enough to figure out what we mean, since when I do a `GET` on a monitoree with a state/local ID, it is being output correctly as an array. But I still think it is probably best to be explicit, since that's the style we use elsewhere, and it will make things more clear if we someday add further `identifier`s.

Reply:
Good call. Since it's a small change I went ahead and just amended my commit https://github.com/SaraAlert/SaraAlert/commit/c6863899da04cdb8c96e7499e12cdb0dd8213ebc